models/Solution.js

```diff
@@ -0,0 +1,9 @@
+class Solution {
+  constructor(userID, solution) {
+    this.postedBy = userID;
+    this.solution = solution;
+    this.postedOn = new Date().toLocaleString();
```

Comment:
Are we allowing the users to update the comments/solutions?

Reply:
No we should just allow solutions to be updated. If comments are to be updated then that would expose user info to the rest api which we don't want.

Reply:
Agree on that.

Reply:
`Solutions` can only be posted by the `Authorities`. And we can expose `authorities` details to the public view. Only `student` data is kept hidden from the api. Since `students` can only comment and not post any solution we can do this.